doc/api/http.md

```diff
@@ -100,11 +117,12 @@ added: v0.3.4
 
 * `options` {Object} Set of configurable options to set on the agent.
   Can have the following fields:
-  * `keepAlive` {Boolean} Keep sockets around in a pool to be used by
-    other requests in the future. Default = `false`
-  * `keepAliveMsecs` {Integer} When using HTTP KeepAlive, how often
-    to send TCP KeepAlive packets over sockets being kept alive.
-    Default = `1000`.  Only relevant if `keepAlive` is set to `true`.
+  * `keepAlive` {Boolean} Keep sockets around even when there are no
+    outstanding requests, so it can be used for future requests without
+    having to reestablish an HTTP connection. Default = `false`
+  * `keepAliveMsecs` {Integer} When using the `keepAlive` option, how
+    often to send TCP `SO_KEEPALIVE` `ACK` packets. Ignored when the
```

Comment:
We should also be consistent with the wording ('packets' vs 'requests' for TCP keepalive), since we use different terminology earlier in the document.